tf_agents/environments/parallel_py_environment_test.py

```diff
@@ -31,19 +32,29 @@
 from tf_agents.specs import array_spec
 
 
+class SlowStartingEnvironment(random_py_environment.RandomPyEnvironment):
+  def __init__(self, *args, time_sleep=0.1, **kwargs):
```

Comment:
We need python 2 support.

```suggestion
  def __init__(self, *args, **kwargs):
    time_sleep = kwargs.pop('time_sleep', 1.0)
     super().__init__(*args, **kwargs)
￼     time.sleep(time_sleep)
```